ioctl/newcmd/account/account.go

```diff
@@ -89,8 +85,8 @@ func NewAccountCmd(client ioctl.Client) *cobra.Command {
 	_flagEndpointUsage, _ := client.SelectTranslation(_flagEndpoint)
 	_flagInsecureUsage, _ := client.SelectTranslation(_flagInsecure)
 
-	ac.PersistentFlags().StringVar(&endpoint, "endpoint", client.Config().Endpoint, _flagEndpointUsage)
-	ac.PersistentFlags().BoolVar(&insecure, "insecure", !client.Config().SecureConnect, _flagInsecureUsage)
+	ac.PersistentFlags().StringVar(client.Endpoint(), "endpoint", client.Config().Endpoint, _flagEndpointUsage)
```

Comment:
add `todo` to move it to maincmd

Reply:
alias/contract/update  these sub commands donot support `--endpoint  --inseure`.  Separately set will be better, i think

Reply:
OK good point